### ISSUE TITLE
Jinja2 templates: add file timestamp as a build string

### ISSUE THREAD

**Issue by anton-malakhov** [conda/conda-build#2509]
Jinja2 does not allow to use `time` and `datetime` python modules by default from meta.yaml. Is there a way to implement something like below?
```
build:
  string: {{ time.ctime(os.path.getmtime("CHANGES")) }}
```
Please add corresponding filters for meta.yaml parser

**Comment by msarahan** [comment]
You can add this at https://github.com/conda/conda-build/blob/master/conda_build/jinja_context.py#L398

PRs welcome.  I don't have time to do this myself right now.

**Comment by anton-malakhov** [comment]
any hint how to make the whole python module available within jinja2 context? e.g. it has access to `os.path` but not to `time` and `datetime`.

**Comment by msarahan** [comment]
# at top of file
import time
import datetime
# in context section that I pointed to:
time=time,
datetime=datetime,

you're just exposing it to the jinja context by giving it the name it should be exposed as, and the python object that should be exposed.

**Comment by msarahan** [comment]
fixed by #2513
